TECHMANIA/Assets/Scripts/Paths.cs

```diff
@@ -109,7 +120,33 @@ public static string GetGameUiSkinRootFolder()
 
     public static string GetGameUiSkinFolder(string name)
     {
-        return Path.Combine(GetGameUiSkinRootFolder(), name);
+        string temp = Path.Combine(GetGameUiSkinRootFolder(), name);
+        return Directory.Exists(temp) ? temp : Path.Combine(GetSaGameUiSkinRootFolder(), name);
+    }
+
+    public static string GetSaSkinFolder()
+    {
+        return Path.Combine(Application.streamingAssetsPath, "Skins");
```

Comment:
These string literals are repeated a few times. Consider making them constants at the top of this class.

Reply:
Did you mean add something like this at the top?
```c#
public const string streamingSkinFolder = Path.Combine(Application.streamingAssetsPath, "Skins");
```

Reply:
```
public const string kSkinFolderName = "Skins";
public const string kNoteSkinFolderName = "Note";
...
```
And then use these constants when you need the strings "Skins" or "Note".